pheweb/serve/data_access/db.py

```diff
@@ -406,6 +406,13 @@ def get_coding(self):
         """
         return
 
+class MissingVariantDB(object):
+    @abc.abstractmethod
+    def get_missing_variant(self, variant: Variant):
+        """Retrieve missing variant data
+        Returns: missing variant results
```

Comment:
Would be great to change the "Returns:" to tell the concrete type of the return value, e.g. is it a class of some sort it returns, or a list of things, or a dictionary of things, or something else? Just so that you don't in fact have to go to the current implementation to know what we are dealing with. Also, type annotations for functions are great, so you could add the return type to the type signature (None OR dict of str keys to Any values, if I read the return type correctly in the implementation):
```python
class MissingVariantDB(object):
    @abc.abstractmethod
    def get_missing_variant(self, variant: Variant) -> Optional[Dict[str,Any]]:
```